CRM/Contact/BAO/SavedSearch.php

```diff
@@ -60,6 +60,11 @@ public static function getFormValues($id) {
     if ($fv) {
       // make sure u CRM_Utils_String::unserialize - since it's stored in serialized form
       $result = CRM_Utils_String::unserialize($fv);
+      if ($result === FALSE) {
+        // See https://github.com/civicrm/civicrm-core/pull/33638
+        Civi::log()->error("Invalid saved_search.form_values for search ID $id. Failed to unserialize. Risk of unexpected search results.");
```

Comment:
It seems to log twice, and also pop up on the screen which surprises me. I guess that's ok but then should have ts() if real users are going to see it. `Civi::log()->error(ts("Invalid saved_search.form_values for search ID %1. Failed to unserialize. Risk of unexpected search results.", [1 => $id]));`

I'll look at doing a status check.

Reply:
The on-screen popup looks like it's because I have the debug setting turned on: https://github.com/civicrm/civicrm-core/blob/c177115fbc58a3f57b6a502b430b2347bb81d18e/CRM/Core/Error/Log.php#L70. So then yes ts() isn't needed because regular users won't see it.

The twice logging is from here https://github.com/civicrm/civicrm-core/blob/c177115fbc58a3f57b6a502b430b2347bb81d18e/CRM/Contact/BAO/GroupContactCache.php#L582-L585 - will leave that for another time.

So I think this is good.

Status check at https://github.com/civicrm/civicrm-core/pull/33672